core/block_svg.js

```diff
@@ -1107,7 +1106,7 @@ Blockly.BlockSvg.prototype.setCommentText = function(text) {
   }
   if (shouldHaveComment) {
     this.commentIcon_ = new Blockly.Comment(this);
```

Comment:
Unrelated to this, but this should have a check if ``Blockly.Comment`` exists.

Reply:
It does, top of that function.

Reply:
Ah yes I see it.